packages/rest/test/integration/rest.application.integration.ts

```diff
@@ -92,6 +92,15 @@ describe('RestApplication (integration)', () => {
       .expect('Hello');
   });
 
+  it('honors basePath', async () => {
+    givenApplication();
+    restApp.basePath('/html');
+    restApp.static('/', ASSETS);
+    await restApp.start();
+    client = createRestAppClient(restApp);
+    await client.get('/html/index.html').expect(200);
```

Comment:
I find this test very confusing. As far as I understand `givenApplication()` helper, it creates an app with no API routes configured. As I understand `app.basePath`, it's intended primarily for customizing the base path of REST APIs.

I see that there are other tests verifying the impact of basePath on different kinds of routes, I agree there is no need to duplicate those tests in `rest.application.integration.ts`.

Can you please add a comment here? Explain that in this test, we are invoking a route for static assets, because basePath applies to all kinds of routes, including static assets.

Alternatively, rework this test to use a handler-function route (see the example below) - I think it will make the intent of this test much easier to understand even without comments.

```ts
app.route('get', '/status', {/*spec*/}, () => {running: true});
// ...
client.get('/api/status').expect(200, {running: true});
```